README.md

```diff
@@ -1,6 +1,6 @@
 # Swagger Assert4J - a Java library for API testing
 
-The Assert4J library lets you test APIs through Java, Groovy or Cucumber. The library has extensive support for REST, SOAP, JDBC and JMS protocols. 
+The Assert4J library lets you test APIs using Java, Groovy or Cucumber. The library has extensive support for REST, SOAP and JDBC protocols. 

```

Comment:
One more comment @olensmar - technically only SOAP is a protocol. REST is more like a convention I suppose, and JDBC is an API (in the classic sense).

Reply:
sure - how about just having "The Assert4J library lets you easily test REST and SOAP services using Java, Groovy or Cucumber." as the first sentence - the JDBC support is there for helping REST/SOAP testing - and not really for JDBC testing in its own right!?

Reply:
Sounds good, as long as we highlight the JDBC support elsewhere.